stable/osg-hosted-ce/osg-hosted-ce/templates/deployment.yaml

```diff
@@ -23,10 +24,6 @@ spec:
         release: {{ .Release.Name }}
         instance: {{ .Values.Instance }}
     spec:
-      {{ if .Values.NodeSelection }}
-      nodeSelector: 
-        kubernetes.io/hostname: {{ .Values.NodeSelection }}
-      {{ end }}
```

Comment:
Do we want to remove this capability? If so, I think it belongs in a separate PR or commit

Reply:
I think we want to keep this.